dev/AppNotifications/AppNotificationContentBuilder/AppNotificationContentBuilder.idl

```diff
@@ -0,0 +1,120 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.Windows.AppNotifications
+{
+    [contract(AppNotificationsContract, 2)]
+    runtimeclass ButtonConfig
+    {
+        ButtonConfig(String content);
+
+        // Gets the text to display on the button.
+        String Content{ get; };
+
+        // Gets app-defined string of arguments that the app can later retrieve once it is activated when the user clicks the button.
+        String Arguments { get; };
```

Comment:
This should be a list of arguments (key / pair values). It's true that the arguments will concatenated into a single string in the final xml but this isn't how the users using the builder will be thinking about it  The users of this API will be adding arguments one at a time thru the AddArgument functions and should retrieve someting in a similar format, that is easy to parse..